ddtrace/tracer.py

```diff
@@ -6,6 +6,7 @@
 from os import environ
 from os import getpid
 import sys
+import threading
```

Comment:
nit: We can leave a note here stating that the threading module needs to be imported before the [ddtrace library clones modules](https://github.com/DataDog/dd-trace-py/blob/29cb7ae6cb042f5b07b751a34e70bf0fcaaf2c0d/ddtrace/bootstrap/sitecustomize.py#L71), something similar to [this](https://github.com/DataDog/dd-trace-py/blob/v2.0.2/ddtrace/__init__.py#L16). This isn't an issue right now because the tracer is one of the first imports but this could change if we refactor the library.